Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButtons.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import "./Button.css";
+import ActionButton from "./ActionButton";
+
+const actionButtons = [
+  {
+    action: "÷"
+  },
+  {
+    action: "x"
+  },
+  {
+    action: "-"
+  },
+  {
+    action: "+"
+  },
+  {
+    action: "="
+  }
+];
+
+function ActionButtons(props) {
+  return (
+    <div className="actionButtons">
+      {actionButtons.map((action, index) => (
```

Comment:
fantastic use of the map function, I like the fact that you build an array of objects and passed the required fields into you. You could also just go with an array to do this however, try to keep your data as simple as possible to get the fastest result.